migrations/2_DEV_local_development.js

```diff
@@ -23,10 +23,13 @@ module.exports = async function (deployer, network, accounts) {
       thresholdAuctionStartUsd: process.env.THRESHOLD_AUCTION_START_USD
     })
 
+    // Deploy DxPriceFeed
+    await deployPriceFeed(deployer)
```

Comment:
While it doesn't much matter what extras are deployed in ganache and I doubt we'll need FixedPriceOracle ioutside ganache, do you think it would be better to add a condition `if (!process.env.USE_FIXED_PRICE_ORACLE)`

Reply:
You are right! I’ll add that 👍🏻